classes/game.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'item'
+class Game < Item
+  attr_accessor :title, :multiplayers, :lastplayed_date
+
+  def initialize(title, publish_date, archived, multiplayers, lastplayed_date)
+    super(nil, publish_date, archived)
+    @title = title
+    @multiplayers = multiplayers
+    @lastplayed_date = lastplayed_date
+  end
+
```

Comment:
To achieve the association between `Item` and `author` classes you should add a new method in item class similar to this;
`def add_author(author)`
    `@author = author`
    `author.add_item(self)`
  `end`